### PR TITLE
Redesign of consent screen

### PR DESCRIPTION
**Pull request by jorissteyn**
Includes feedback from #456 and testing on test2:

 - styled for smaller screens, tested in many browsers
 - removed IE8 support

Todo:

 - [x] Fix failing SSO test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,17 @@ In short, themes require compilation which can be done by running the following
     (cd theme && npm install && grunt)
 ```
 
+To setup the required tooling on the VM, the following steps might be useful:
```
Comment (MKodde) [inline README.md]: I've updated the installation instructions slightly.

File: languages/en.php [hunk 2]
```diff
@@ -180,6 +180,44 @@
     'consent_footer_text_plural'              => 'You are using %1$s services via SURFconext. <a href="https://profile.surfconext.nl/" target="_blank">View the list of services and your profile information.</a>',
     'consent_footer_text_first_consent'       => 'You are not using any services via SURFconext. <a href="https://profile.surfconext.nl/" target="_blank">View your profile information.</a>',
 
+    // Consent slidein: Kloppen de getoonde gegevens niet?
+    'consent_slidein_correction_title' => 'Kloppen de getoonde gegevens niet?',
```
Comment (MKodde) [inline languages/en.php]: Are these texts to be translated?

File: theme/material/stylesheets/_vars.css.sass [hunk 3]
```diff
@@ -1,6 +1,7 @@
 
 // Colors
 $blue: #4DB3CF
+$dark-blue: #017499
```
Comment (MKodde) [inline theme/material/stylesheets/_vars.css.sass]: Lovin' this color, that slightly darker blue hits the spot!

**Comment by jorissteyn** [comment]
Thanks! I've fixed the failing SSO test and prevented the page from scrolling to the top when opening a slide-in.

**Pull request merged**